### DISCUSSION TITLE
part-ii/2011-84/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#414] [Question comments]
Part II 2011 Logic and Set Theory: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2011-84/

**Comment by ArthurConmy** [comment]
<details>
<summary>iii)</summary>

No consider $\alpha = \omega^3$ and Cantor Normal Form.
</details> 

<details>
<summary>iv)</summary>

No consider $\alpha = 2\omega_1$ and we get $\omega_1^2$ and $2 \omega_1^2$.
</details>
